config/routes.rb

```diff
@@ -35,6 +35,9 @@
 
     resources :preferred_facilities, only: %i[index create destroy]
 
+    resources :apps, only: %i[index show]
+    get 'apps/scopes/:category', to: 'apps#scopes', defaults: { category: 'unknown_category' }
+
```

Comment:
this explicit routing is necessary so that `apps/scopes` isnt interpreted as a call to `#show`